src/AutoComplete/AutoComplete.js

```diff
@@ -113,6 +113,10 @@ class AutoComplete extends Component {
      */
     menuCloseDelay: PropTypes.number,
     /**
+    * Override style for menuItem
```

Comment:
Please could you use the standard wording here? "Override the inline-styles of the menuItem element."

If you could fix-up the wording on the other style props in this component too, that would be great!